person-service/src/main/java/gov/cdc/etldatapipeline/person/model/dto/persondetail/Phone.java

```diff
@@ -23,6 +23,8 @@ public class Phone implements ExtendPerson {
     private String cd;
     @JsonProperty("ph_tl_uid")
     private Long teleLocatorUid;
+    @JsonProperty("phone_comments")
```

Comment:
Optional: JsonProperty might not be required in this case since we use the Gson CamelCase library call transformer to automatically convert. We can use JsonProperty if the incoming and the transformed names are totally different.

We can still use the JsonProperty here. No Harm!

Reply:
Right! Let's make it consistent.